apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -2256,13 +2342,29 @@ static void libc_stdio_rawoutstream_tc(void)
 
 	struct lib_rawoutstream_s rawoutstream;
 
+	/* Negative case, no character is put to the rawoutstream */
+
 	fd = open(filename, O_RDONLY);
 	TC_ASSERT_GEQ("open", fd, 0);
 
 	lib_rawoutstream((FAR struct lib_rawoutstream_s *)&rawoutstream, fd);
+	rawoutstream.public.put(&rawoutstream.public, 1);
+	TC_ASSERT_EQ_CLEANUP("rawoutstream_putc", rawoutstream.public.nput, 0, close(fd));
+	TC_ASSERT_EQ_CLEANUP("lib_rawoutstream", rawoutstream.fd, fd, close(fd));
```

Comment:
I think this will be next to lib_rawoutstream((FAR struct lib_rawoutstream_s *)&rawoutstream, fd);